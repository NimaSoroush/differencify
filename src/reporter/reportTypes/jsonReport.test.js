import getJsonReport from './jsonReport';

const results = [
  {
    testName: 'default',
    result: 'no mismatch found',
  },
];

describe('JSON report', () => {
  it('generates a JSON report', () => {
    const report = getJsonReport(results);
    expect(report).toMatchSnapshot();
  });
});
