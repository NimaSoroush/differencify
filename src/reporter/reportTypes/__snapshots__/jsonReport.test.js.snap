// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JSON report generates a JSON report 1`] = `"[{\\"testName\\":\\"default\\",\\"result\\":\\"no mismatch found\\"}]"`;
